config-model/src/main/java/com/yahoo/vespa/model/container/ContainerCluster.java

```diff
@@ -479,7 +480,8 @@ public void setDocumentApi(ContainerDocumentApi containerDocumentApi) {
     @NonNull
     public DocprocChains getDocprocChains() {
         if (containerDocproc == null)
-            throw new IllegalStateException("Null docproc components!");
+            throw new IllegalStateException("Search components not found in container cluster '" + getSubId() +
```

Comment:
Copy/paste error. I think you mean 'document-processing'.

Reply:
Fixed - thanks!